website/docs/docs/cloud-integrations/downstream-exposures-tableau.md

```diff
@@ -0,0 +1,107 @@
+image: /img/docs/cloud-integrations/auto-exposures/explorer-lineage2.jpg
+---
+
+# Configure exposures automatically <Lifecycle status="enterprise" />
+
+<IntroText>
+Configure and automatically populate downstream exposures for supported BI tool integrations. Visualize and orchestrate them through <a href="https://docs.getdbt.com/docs/collaborate/explore-projects">dbt Explorer</a> and the [dbt Cloud job scheduler](/docs/deploy/job-scheduler) for a richer experience.
+
+</IntroText>
+
+As a data team, it’s critical that you have context into the downstream use cases and users of your data products. In dbt Cloud, you can configure downstream exposures in two ways:
+- Manual &mdash; Defined [manually](/docs/build/exposures#declaring-an-exposure) and explicitly in your project’s YAML files.
+- Automatic &mdash; Pulled automatically for supported dbt Cloud integrations. dbt Cloud automatically creates and visualizes downstream exposures, removing the need for manual YAML definitions. These downstream exposures are stored in dbt’s metadata system, appear in [dbt Explorer](/docs/collaborate/explore-projects), and behave like manual exposures, however they don’t exist in YAML files.
+
+By leveraging downstream [exposures](/docs/build/exposures) automatically, data teams can:
+
+- Gain a better understanding of how models are used in downstream analytics, improving governance and decision-making.
+- Reduce incidents and optimize workflows by linking upstream models to downstream dependencies.
+- Automate exposure tracking for supported BI tools, ensuring lineage is always up to date.
+- [Orchestrate exposures](/docs/cloud-integrations/orchestrate-exposures) to refresh the underlying data sources during scheduled dbt jobs, improving timeliness and reducing costs. Orchestrating exposures is essentially a way to ensure that your BI tools are updated regularly by using the [dbt Cloud job scheduler](/docs/deploy/job-scheduler). See the [previous page](/docs/cloud-integrations/downstream-exposures) for more info.
+
+:::info Tableau Server
+If you're using Tableau Server, you need to [allowlist dbt Cloud's IP addresses](/docs/cloud/about-cloud/access-regions-ip-addresses) for your dbt Cloud region.
+:::
+
+## Prerequisites
+
+To configure downstream exposures automatically, you should meet the following:
+
+1. Your environment and jobs are on a supported [release track](/docs/dbt-versions/cloud-release-tracks) dbt.
+2. You have a dbt Cloud account on the [Enterprise plan](https://www.getdbt.com/pricing/).
+3. You have set up a [production](/docs/deploy/deploy-environments#set-as-production-environment) deployment environment for each project you want to explore, with at least one successful job run. 
+4. You have [admin permissions](/docs/cloud/manage-access/enterprise-permissions) in dbt Cloud to edit project settings or production environment settings.
```

Comment:
```suggestion
4. You have [proper permissions](/docs/cloud/manage-access/enterprise-permissions) to edit dbt Cloud project or production environment settings.
```

Reply:
ok so double checking this [here](https://dbt-labs.slack.com/archives/C04N364M8QY/p1742205410421569) to ensure it can be multiple diff types of permissions so long as they can edit.